templates/partials/livestream-notification.html

```diff
@@ -0,0 +1,10 @@
+{% if livestream %}
+    <div class="col-12">
+        <div class="p-notification--information p-snapcraft-nps-banner">
+            <p class="p-notification__response">
+                <span class="p-notification__status">📺 LIVE</span>
+                <a href="{{ livestream.url }}" target="_blank" class="p-link--external">Watch our Snapcraft Live Stream </a>&nbsp;{{ livestream.text }}
```

Comment:
Maybe we can give alternative messages for cool down and interval but for now this serves the purpose!